x-pack/plugins/maps/public/shared/layers/sources/es_search_source/es_search_source.js

```diff
@@ -253,10 +336,30 @@ export class ESSearchSource extends AbstractESSource {
   getSourceTooltipContent(sourceDataRequest) {
     const featureCollection = sourceDataRequest ? sourceDataRequest.getData() : null;
     const meta = sourceDataRequest ? sourceDataRequest.getMeta() : {};
```

Comment:
Looks like `getMeta()` always minimally returns an empty object so no need for this conditional assignment

Reply:
The conditional is for whether sourceDataRequest exists. Can not call `getMeta` on null. This statement is still needed